doc/classes/XRInterface.xml

```diff
@@ -141,6 +141,7 @@
 			<param index="0" name="mode" type="int" enum="XRInterface.PlayAreaMode" />
 			<description>
 				Sets the active play area mode, will return [code]false[/code] if the mode can't be used with this interface.
+				Changing this after the interface has already been initialized can be jarring for the player, so it's recommended to recenter on the HMD or make the switch during a scene change.
```

Comment:
This kind of statement usually fits snuggly in a note. I also assume that "recenter" here refers to a method.
```suggestion
				[b]Note:[/b] Changing this after the interface has already been initialized can be jarring for the player, so it's recommended to recenter on the HMD with [method XRServer.center_on_hmd] or make the switch during a scene change.
```

Reply:
Thanks! Included in my latest push (with a little more detail about when it makes sense to recenter on the HMD)